.github/PULL_REQUEST_TEMPLATE/RELEASE_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
 Link to the milestone on Github, example: https://github.com/nimblehq/android-templates/milestone/1?closed=1
 
 ## Features
-Provide the IDs of the issues in each section (feature, chore and bug), example:
+Provide the issue numbers in each section (feature, chore and bug), example:
```

Comment:
I think it should be `issue IDs` instead of `issue numbers` https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#list-issues-assigned-to-the-authenticated-user. `Number` would be confusing here.

Reply:
GitHub calls it `number` sometimes, nvm 👍 https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword